src/lib.rs

```diff
@@ -39,4 +39,9 @@ pub mod nss {
     pub const CKO_NSS: CK_ULONG = CKO_VENDOR_DEFINED | NSSCK_VENDOR_NSS;
     pub const CKO_NSS_TRUST: CK_ULONG = CKO_NSS + 3;
     pub const CKO_NSS_BUILTIN_ROOT_LIST: CK_ULONG = CKO_NSS + 4;
+
```

Comment:
It might be nice to have a brief explanation of the situation here (basically what you wrote in the PR conversation tab).